BookingsAPI/Bookings.IntegrationTests/Features/Persons.feature

```diff
@@ -63,4 +63,10 @@ Feature: Persons
     Given I have a get person suitability answers by username request with an valid username
     When I send the request to the endpoint
     Then the response should have the status OK and success status True
-    And persons suitability answers should be retrieved
\ No newline at end of file
+    And 'with' suitability answers should be retrieved
+
+  Scenario: Get person without suitability answers for valid username
```

Comment:
This is a bit hard to understand exactly what it's testing. Could it be rewritten so that you get a list of suitability answers or an `empty` list a suitability answers? (if that's what you're testing that is)